javascript/javascript1/week3/whenwillbethere.js

```diff
@@ -0,0 +1,17 @@
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function calculateTravelTime(travelInformation) {
```

Comment:
This is partially correct, and partially wrong. I think choosing an easier logic could help with avoiding mistakes.